std/algorithm/iteration.d

```diff
@@ -2284,20 +2346,6 @@ if (isInputRange!Range)
     ]));
 }
 
-version (none) // this example requires support for non-equivalence relations
```

Comment:
It'd be better instead to keep this and assert it fails.

Reply:
You mean duplicate it? I didn't remove it, just moved it under `splitBy` and changed to use it instead.

Reply:
Tried to keep the test-fail case, but it failed by triggering an implementation-specific assert:

```
    this(RefCounted!(OuterRange) origin)
    {
        groupNum = origin.groupNum;
        current = origin.current.save;
        assert(!current.empty, "Passed range 'r' must not be empty");
        static if (eqEquivalenceAssured)
        {
            start = origin.current.save;

            // Check for reflexivity.
            assert(eq(start.front, current.front), // <------ this failed
                "predicate is not reflexive");
        }

        mothership = origin;
    }
```

Testing for that would be quite fragile in my opinion.